torch/nn/quantized/modules/utils.py

```diff
@@ -1,13 +1,13 @@
 import torch
 
 def _quantize_weight(float_wt, observer):
+    wt_scale, wt_zp = observer.calculate_qparams()
```

Comment:
Would it be cleaner to do:
```python
q_params = observer.get_qparams()
if ....
    qweight = torch.quantize_per_tensor(float_wt, float(q_params[0]), int(q_params[1]), torch.qint8)
elif ...
    qweight = torch.quantize_per_tensor(float_wt, q_params[0].to(torch.double), q_params[1].to(torch.int64), q_params[2], torch.qint8)
```

Reply:
probably better to keep this way... I think maybe we don't need get_qparams, we can get access to ch_axis through observer module